From 412436b9d72ba24464243b659c09ec507fb2be33 Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Sun, 27 Sep 2020 11:09:21 +0800
Subject: [PATCH] gpio: rockchip: make gpio-bank name with bank index

Fix the name of gpio bank to be more general, make it with bank index
instead of node name.

Change-Id: Ia7e61e2eaa2cdd8f3ff0482cfc7c857f2a505935
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 drivers/gpio/gpio-rockchip.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpio/gpio-rockchip.c b/drivers/gpio/gpio-rockchip.c
index 69c5deae2993..fa924d7b9f58 100644
--- a/drivers/gpio/gpio-rockchip.c
+++ b/drivers/gpio/gpio-rockchip.c
@@ -706,8 +706,8 @@ static int rockchip_gpio_probe(struct platform_device *pdev)
 	if (id < 0)
 		id = gpio++;
 
-	bank->name = devm_kasprintf(dev, GFP_KERNEL, "%pOF", np);
 	bank->pin_base = id * GPIO_BANK_PIN_NUM;
+	bank->name = devm_kasprintf(dev, GFP_KERNEL, "GPIO%d", id);
 	bank->nr_pins = GPIO_BANK_PIN_NUM;
 	bank->dev = dev;
 	bank->of_node = dev->of_node;
-- 
2.35.3

